app/models/ex_link.rb

```diff
@@ -1,3 +1,23 @@
 class ExLink < ActiveRecord::Base
   validates :label, :url, presence: true
+  before_save :polish_tags
+
+  # TODO: Adding image based on feature requests from David.
+  # TODO: This is just a mock-up
+  has_attached_file :image,
+                    styles: { medium: "300x300>", thumb: "100x100>" },
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.